crates/bevy_pbr/src/wireframe.rs

```diff
@@ -117,8 +118,7 @@ fn queue_wireframes(
         .unwrap();
     let msaa_key = MeshPipelineKey::from_msaa_samples(msaa.samples);
     for (view, visible_entities, mut opaque_phase) in views.iter_mut() {
-        let view_matrix = view.transform.compute_matrix();
-        let view_row_2 = view_matrix.row(2);
+        let rangefinder = ViewRangefinder3d::from_view(view);
```

Comment:
It would be nice if this was a method on `View` instead like `view.rangefinder()`.
That way it does not require an import and is more discover-able.

Reply:
I like that suggestion a lot!

Reply:
I've adopted this change.